src/main/kotlin/org/wycliffeassociates/otter/jvm/app/ui/projecthome/ViewModel/ProjectHomeViewModel.kt

```diff
@@ -0,0 +1,15 @@
+package org.wycliffeassociates.otter.jvm.app.ui.projecthome.ViewModel
+
+import org.wycliffeassociates.otter.common.data.dao.Dao
+import org.wycliffeassociates.otter.common.data.model.Project
+import org.wycliffeassociates.otter.common.domain.GetProjectsUseCase
+import tornadofx.*
+import org.wycliffeassociates.otter.common.data.model.Language
+import io.reactivex.Observable
+import java.util.*
+
+class ProjectHomeViewModel: ViewModel() {
+    // TODO(need repo)?
+    val getProjectsUseCase = GetProjectsUseCase()
+    val projects : List<Project> = getProjectsUseCase.getProjects().toList().blockingGet().flatMap{it}
```

Comment:
blocking get defeats the purpose of not blocking on the UI thread. 

You need to put this in an ObservableList, and bind that to the datagrid

When the list is populated because the observable has emitted, you update the ObservableList and datagrid will automatically update the view because of the observable binding